tests/test_cli.py

```diff
@@ -77,8 +75,6 @@ def test_selection(capfd):
         )
 
         assert "model_with_scripts" not in captured.out
-        assert "model_feature_store" in captured.out
-        assert "model_empty_scripts" in captured.out
```

Comment:
why are you removing assertions?

Reply:
These are actually not supposed to be printed as there are no scripts associated with them.

Reply:
Then change the check to `not in captured.out`, so we make sure they are indeed not printed